src/features/user/DonationReceipt/DonationReceiptWrapper.tsx

```diff
@@ -82,25 +96,25 @@ const DonationReceiptWrapper = () => {
         };
         response =
           addressGuid === null
-            ? await putApi<IssuedReceiptDataApi>(
+            ? await putApi<IssuedReceiptDataApi, ReceiptVerificationPayload>(
                 '/app/donationReceipt/verify',
-                payload
+                { payload }
               )
-            : await putApiAuthenticated<IssuedReceiptDataApi>(
-                '/app/donationReceipt/verify',
-                payload
-              );
+            : await putApiAuthenticated<
+                IssuedReceiptDataApi,
+                ReceiptVerificationPayload
+              >('/app/donationReceipt/verify', { payload });
       } else if (operation === RECEIPT_STATUS.ISSUE) {
-        const payload = {
+        const payload: DonationReceiptPayload = {
           receiptAddress: addressGuid,
           donationUids: donationUids.join(','),
           verificationDate,
         };
 
-        response = await postApiAuthenticated<IssuedReceiptDataApi>(
-          '/app/donationReceipts',
-          payload
-        );
+        response = await postApiAuthenticated<
+          IssuedReceiptDataApi,
+          DonationReceiptPayload
```

Comment:
What are we doing here? The type name does not make that clear.

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/2455/commits/ffda4656f0efdd5819edd7971938df84ac89b8b0